DS/DSAssignment5/LinkListInserction.java

```diff
@@ -0,0 +1,25 @@
+import java.util.LinkedList;
+import java.util.List;
+
+public class LinkListInserction {
+
+	public List<Integer> list = new LinkedList<Integer>();
```

Comment:
ACTION: Why use java collection? I don't think the whole program is of any use if collection is used.